monitors/command_test.go

```diff
@@ -200,3 +200,36 @@ func TestDiffMonitorsWithScopes(t *testing.T) {
 		t.Errorf("expected:\n%s\n, output:\n%s\n", expected, diff)
 	}
 }
+
+func TestMonitorLoadRules(t *testing.T) {
```

Comment:
for example,

```suggestion
func TestMonitorLoadRulesWithBOM(t *testing.T) {
```

because this function tests if it ignore BOM.